### ISSUE TITLE
Automatic Drive Letter Selection in Windows

### ISSUE THREAD

**Issue by deusexvmachina** [keybase/client#3195]
Keybase GUI Version: 1.0.16-20160622134333+f8bd1e6

Windows 7 64-bit, keybase client on installation chose a drive letter already assigned to another physical hard drive, rendering the KBFS mapping invisible and unusable.  As the client is so new, my particular issue was difficult to pin down as an issue at all as I was unaware of how it was supposed to correctly function.

**Comment by zanderz** [comment]
Yes, we still need to do something differently here. An upcoming build will allow a workaround at least - we'll keep you posted.

**Comment by zanderz** [comment]
Hi, the version made available this morning, 1.0.16-20160629093029+dc3f5a6, will let you set the drive letter with an environment variable KEYBASE_DRIVE_LETTER, which needs to include a colon. Set this globally using control panel and restart and your specified letter should be mounted.

**Comment by zanderz** [comment]
Sorry for the confusion, but the environment variable was renamed to KEYBASE_MOUNTDIR. Note that there is also a new command, keybase kbfsmount set X:. Either one should work.
